packaging/linux/post_install.sh

```diff
@@ -39,9 +39,11 @@ if [ -z "$currlink" ] ; then
         if killall kbfsfuse &> /dev/null ; then
             echo Shutting down kbfsfuse...
         fi
-        rmdir "$rootlink"
         echo You must run run_keybase to restore file system access.
     fi
+    if rmdir "$rootlink" &> /dev/null ; then
+        echo Replacing old $rootlink directory.
+    fi
     if ln -s "$mount1" "$rootlink" &> /dev/null ; then
```

Comment:
Let's add the `-T` flag to `ln`. That guarantees that we try to create a link with exactly the name `/keybase`, rather than implicitly creating one _inside_ that directory (with some name inferred from its target) if that directory still exists when we get here.

Reply:
Oh cool, will do.